src/components/Button/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ComponentPropsWithoutRef } from "react";
+
+export interface ButtonProps extends ComponentPropsWithoutRef<"button"> {
+  children?: React.ReactNode;
+  iconStart?: JSX.Element;
+  iconEnd?: JSX.Element;
+}
+
+export function Button(props: ButtonProps) {
+  const hoverClass = "hover:bg-tertiaryAlt";
+  const activeClass = "active:bg-secondaryAlt";
+  const disabledClass =
+    "disabled:bg-[#DADADA] disabled:cursor-not-allowed disabled:text-white";
+  const paddingClass = "px-4 py-3";
+  const sizeClass = "w-full h-[48px]";
```

Comment:
Let's avoid custom values like: h-[48px]. You can replace by: `h-12`.  BTW if you remove the w-full isn't take the whole place by default!?